fraud-detection-new-isoscape/origin_validation.py

```diff
@@ -0,0 +1,128 @@
+# Import Libraries
```

Comment:
why do you need a new file here for new-isoscape?  is it busted?  I thought new-isoscape was calling the other function so you dont need this.

Reply:
yes that's true, File will be deleted in the coming commit